.docker/app/Dockerfile

```diff
@@ -17,13 +17,13 @@ USER root
 # libc6-compat is added to allow m1 build of nokogiri
 RUN apk add -U --no-cache \
       bash=5.1.16-r2 \
-      busybox=1.35.0-r16 \      
```

Comment:
@cdoyle-temple Are busybox and ssl_client required here?  Just wondering if we need to define them here since they are part of the base image.

Reply:
I don't know. Should I try without them?

Reply:
Apparently not!